README.md

```diff
@@ -317,6 +317,51 @@ pipenv run python gatorgrouper_cli.py --file filepath --method graph
 --objective-weights LIST
 ```
 
+### Student Data Files
+
+Gatorgrouper can import student data from a supplied CSV file using the
+`--file data.csv` flag. Similarly, data about student's gropuing preferences
+can be imported via `--preferences preferences.csv`.
+
+The student data file contains one line for each student, along with a
+comma-separated list of data points for that student. For example,
+a survey asking for name, willingness to be a leader,
+years of programming experience,
+and preferred language might yield the following line in the CSV file:
+
+```
+Bob,True,1.0,Python
```

Comment:
When I tried this input in the tool it did not run. Am I missing something?

Reply:
Did you have any other lines in your CSV file? The tool doesn't run with only one student, but copy-and-pasting that line and changing the names worked on my machine. Also, if using the graph algorithm, you may have to specify the objective measures in order to avoid errors about converting strings to floats